src/chapters/http/how-the-internet-works.rst

```diff
@@ -0,0 +1,247 @@
+
+   * - TCP/IP
+     - Transmission Control Protocol / Internet Protocol
+     - Low-level web communication for transferring small chunks of raw data known as *packets*
+   * - DNS
+     - Domain Name Service
+     - Translates human-friendly names into server addresses
+
+A thorough understanding of each of these protocols is well beyond the scope of this class. However, as a web developer it is important you have a general understanding of their roles. Each protocol has a different and critical job in enabling web communication.
+
+HTTP
+^^^^
+
+.. index:: ! HTTP, ! HTTPS
+
+**HTTP** is the most important protocol for web developers to understand, which you may have guessed from the title of this chapter. It specifies how requests for common web data---such as HTML files or images---should be structured, as well as responses to such requests. The details of request and response message structure are the topic of the rest of this chapter. 
+
+**HTTPS** refers to the HTTP protocol used with a secure connection. A secure connection encrypts so that it can't be read while in-transit. The data is encrypted by the server/client before being transmitted, and decrypted once it is received by the client/server. The precise details of how such encryption works is beyond the scope of this course. 
+
+TCP/IP
+^^^^^^
+
+.. index:: ! TCP/IP
+
+**TCP/IP** is a low-level protocol that is quite complicated. For our purposes, it is important only to know that TCP/IP is the standard that allows *raw data* to get from one place to another on the Internet. 
+
+When a server sends a file back to a client, that file must physically be sent across a series of network components, including cables, routers, and switches. Files are broken down into *packets*---small chunks of a standard size---that are individually sent from one location to the next, until arriving at their final destination and being reassembled.
+
+.. admonition:: Fun Fact
+
+   You can think of the Internet as a `"series of tubes." <https://www.youtube.com/watch?time_continue=15&v=_cZC67wXUTs>`_ This phrase was used by a U.S. Senator in 2006 and widely mocked. However, we think it's actually a reasonable analogy. TCP/IP allows data to be passed from one tube to another until reaching the final destination.
+
+DNS
+^^^
+
+.. index:: ! DNS, ! IP address
+
+**DNS** is the address book of the Internet. It enables us to use readable and memorable names for servers, such as ``www.launchcode.org`` or ``mail.google.com``. Such names are called **domain names**, and they function as aliases for the actual server addresses.
+
+Every server on the internet has a numerical address known as an **IP address**. When a message is addressed using a domain name, the corresponding IP address must be determined before it can be sent. 
+
+.. admonition:: Example
+
+   The IP addresses of ``www.launchcode.org`` and ``mail.google.com`` are ``104.25.127.113`` and ``172.217.5.229``, respectively.
+
```

Comment:
So I got different IP address for launchcode: 104.26.14.63 and mail.google.com: 142.251.33.197 when using the below MX Toolbox.  Not sure what that is about.

 I was trying to see if I could just type the IP in the address bar and get to the website, using http://IP address - works for google (not for LC, Direct IP access not allowed due to cloud flare network)